local/setup.sh

```diff
@@ -1,8 +1,45 @@
 #!/usr/bin/env bash
-
-set -o errexit
```

Comment:
`set -e` comes with some pretty major drawbacks. I've found that it's generally better to have a `die` function that you call when running an important command. [This wiki](https://mywiki.wooledge.org/BashFAQ/105) does a great job of explaining why `set -e` is problematic. It's very likely that `errexit`, `nounset`, or `pipefail` were responsible for some of the flakiness I was seeing.